rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/220_filters_bucket.yml

```diff
@@ -137,6 +137,40 @@ setup:
   - match: { aggregations.the_filter.buckets.1.doc_count: 1 }
   - match: { aggregations.the_filter.buckets.1.the_avg.value: 151.0 }
 
+---
+"named filters with bucket sort test":
+  - do:
+      search:
+        rest_total_hits_as_int: true
+        body:
+          size: 0
+          aggs:
+            the_filter:
+              filters:
+                keyed_bucket_in_array: true
```

Comment:
I don't particularly like the name of this parameter. But I don't really know of a better name. @not-napoleon, do you have an opinion here?